README.md

```diff
@@ -252,6 +252,27 @@ exit without further processing.
 - If a file has already been uploaded, the process will exit without further
 processing. To overwrite existing files, use the `-force-overwrite` flag.
 
+### Resume a multiple file upload
+
+It is possible to resume an interrupted upload from the last not uploaded file by adding the `-continue` flag which will prevent the `sda-cli` to return an error if a file is already uploaded on the target directory. For example:
+
+```bash
+./sda-cli -config <configuration_file> upload <encrypted_file_to_re-upload> <encrypted_file_to_upload> -continue
+```
```

Comment:
```suggestion
Suppose you run the following command to upload two encrypted files

```bash
./sda-cli -config <configuration_file> upload <encrypted_file_to_upload_1> <encrypted_file_to_upload_2>
```

After `<encrypted_file_to_upload_1>` has been uploaded, the process is interrupted while uploading `<encrypted_file_to_upload_2>`.  To resume the upload, simply add the `-continue` flag to the command, as shown below:

```bash
./sda-cli -config <configuration_file> upload <encrypted_file_to_re-upload> <encrypted_file_to_upload> -continue
```
```

Reply:
Nice explanation! I replaced the confusing part in the README with the one above.